import-export-cli/go.mod

```diff
@@ -20,8 +20,12 @@ require (
 	github.com/wso2/k8s-api-operator/api-operator v0.0.0-20210125090919-4d9482a2f6b8
 	golang.org/x/crypto v0.0.0-20200414173820-0848c9571904
 	gopkg.in/yaml.v2 v2.3.0
+	k8s.io/api v0.18.2
+	sigs.k8s.io/yaml v1.2.0 // indirect
 )
 
 replace k8s.io/client-go => k8s.io/client-go v0.18.2
 
+replace github.com/wso2/k8s-api-operator/api-operator => /home/jayanie/Documents/apim-4-operator-feature/k8s-api-operator-new-code/k8s-api-operator/api-operator
```

Comment:
Update API operator version

Reply:
+1, Let's update this before sending alpha

Reply:
Updated with c8e04729f743adeabf71a0e3032576a66d771f5e